srcpkgs/neovide/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'neovide'
+pkgname=neovide
+version=0.13.3
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config python3 cmake curl"
+makedepends="fontconfig-devel freetype-devel libxcb-devel libX11-devel gtk+3-devel libglvnd-devel sndio-devel openssl-devel libcxx-devel neovim"
+depends="neovim gtk+3"
+short_desc="No Nonsense Neovim Client in Rust"
+maintainer="Stefan Weigl-Bosker <stefan@s00.xyz>"
+license="MIT"
+homepage="neovide.dev"
+changelog="https://github.com/neovide/neovide/compare/0.13.2...0.13.3"
```

Comment:
That's not a good idea, hardcoding the previous version for the changelog is undesirable. The link should instead be https://github.com/neovide/neovide/releases.